packages/contracts-bedrock/justfile

```diff
@@ -29,6 +29,10 @@ test-kontrol-no-build:
 test-rerun: build-go-ffi
   forge test --rerun -vvv
 
+# Run extra fuzz iterations for new fuzz tests.
+test-extra-fuzz-runs: build-go-ffi
+  ./scripts/testing/test-extra-fuzz-runs.sh
```

Comment:
Non-blocker and don't want to bikeshed on the name too much, but it is more nuanced than just `extra-fuzz-runs` so would be nice if we can convey that. Maybe one of these?
```suggestion
# Run extra fuzz iterations for new fuzz tests.
test-heavy-fuzz-new-tests: build-go-ffi
  ./scripts/testing/test-heavy-fuzz-new-tests.sh
```
```suggestion
# Run extra fuzz iterations for new fuzz tests.
test-fuzz-new-tests: build-go-ffi
  ./scripts/testing/test-fuzz-new-tests.sh
```

Reply:
used `heavy-fuzz-new-tests`